pom.xml

```diff
@@ -467,8 +467,14 @@
 
       <dependency>
         <groupId>com.google.code.findbugs</groupId>
+        <artifactId>annotations</artifactId>
```

Comment:
This is the parent `pom.xml`, which doesn't have dependencies of its own. These are declared for consistent use in sub-modules. We use this dependency in `java-sdk-all`.